code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -269,16 +269,24 @@
 			return
 	//SKYRAT EDIT ADDITION BEGIN - EMOTES
 	if(zone_selected == BODY_ZONE_PRECISE_GROIN && target.dir == src.dir)
-		if(HAS_TRAIT(target, TRAIT_IRONASS))
-			var/obj/item/bodypart/affecting = src.get_bodypart("[(src.active_hand_index % 2 == 0) ? "r" : "l" ]_arm")
-			if(affecting?.receive_damage(2))
-				src.update_damage_overlays()
-			visible_message("<span class='danger'>[src] tried slapping [target]'s ass, however it was much harder than expected!</span>",
-			"<span class='danger'>You tried slapping [target]'s ass, but it felt like metal, ouch!</span>",\
-			"You hear a sore sounding slap.", ignored_mobs = list(target))
-			playsound(target.loc, 'sound/effects/snap.ogg', 50, TRUE, -1)
-			to_chat(target, "<span class='danger'>[src] tried slapping your ass, but it was deflected!")
-			return
+		if(HAS_TRAIT(target, TRAIT_PERSONALSPACE) && (target.stat != UNCONSCIOUS) && (!target.handcuffed)) //You need to be conscious and uncuffed to use Personal Space
+			if(target.combat_mode && (!HAS_TRAIT(target, TRAIT_PACIFISM))) //Being pacified prevents violent counters
+				var/obj/item/bodypart/affecting = src.get_bodypart(BODY_ZONE_HEAD)
+				if(affecting?.receive_damage(2))
```

Comment:
What does the `2` mean here?

Reply:
The 2 is the amount of damage dealt to the person's head. I'll change this to a constant.

Reply:
Actually, on second thought - as it's only used here, would it make more sense to just add a comment saying what it is?
I understand the reasons behind not using magic numbers, but this one is only used here.

Reply:
It's still good practice, if it's only used there, make a define at the start of the file and then do `#undef DEFINE_NAME` at the end of the file, so it doesn't pollute the global namespace.

Reply:
I see, I'll do that.